docs/lib/restapi/fragments/flutter/getting-started/30_initapi.md

```diff
@@ -0,0 +1,24 @@
+To initialize the Amplify API category you call `Amplify.addPlugin()` method for each category. To complete initialization call `Amplify.configure()`.
+
+Your code should look like this:
+
+```dart
+import 'package:amplify_flutter/amplify.dart';
+import 'package:amplify_api/amplify_api.dart';
+
+import 'amplifyconfiguration.dart';
+
+class MyAmplifyApp extends StatefulWidget {
+
+    Amplify amplifyInstance = Amplify();
+
+    @override
+    void initState() {
+        super.initState();
+
+        AmplifyAPI api = AmplifyAPI();
+        Amplify.addPlugin(apiRest);
```

Comment:
this doesn't look right. Are these code samples tested?

Reply:
Thank you, I made a mistake when updating (should have just copy pasted entire code instead of individual line changes)